### DISCUSSION TITLE
Get block position

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#974] [Help] [answered]
Hello,
is it possible to get the position of the block without any events? like youre getting the block out of a list.
~jochem

**Comment by MrGazdag** [comment, marked as answer]
A Block object has no references to being in a world, a Block object just contains Block type related information. Events contain other data, like positiom. If you would like to store certain blocks like this, the create your own storage object for this and store those, as a raw Block instance has no ties to that kind of data.
